src/Microsoft.ML.Featurizers/DateTimeTransformer.cs

```diff
@@ -148,7 +148,7 @@ internal DateTimeEstimator(IHostEnvironment env, Options options)
 
         public DateTimeTransformer Fit(IDataView input)
         {
-            return new DateTimeTransformer(_host, _options.Source, _options.Prefix, _options.Country);
+            return new DateTimeTransformer(_host, _options.Source, _options.Prefix, _options.Country, input);
         }
```

Comment:
Nit: Since you are using only Schema in your class, would it be better to pass in the schema instead of input?

Reply:
Good idea. Changed it to take just the schema.